test/e2e/testsuites/dynamically_provisioned_resize_volume_tester.go

```diff
@@ -47,6 +48,7 @@ type DynamicallyProvisionedResizeVolumeTest struct {
 	StorageClassParameters map[string]string
 	Pod                    PodDetails
 	Volume                 VolumeDetails
+	DetachVM               bool
```

Comment:
Consider `ResizeOffline` to better reflect the scenario. The VM isn't what's being detached here.